src/osd/OSDMap.cc

```diff
@@ -1364,7 +1364,6 @@ void OSDMap::get_blocklist(std::set<entity_addr_t> *bl) const
 
 void OSDMap::set_max_osd(int m)
 {
-  int o = max_osd;
```

Comment:
i also have this change in #37021. will drop it from my PR if your change gets merged earlier than mine.